spec/rubykassa/xml_interface_spec.rb

```diff
@@ -31,4 +31,42 @@
   it "correctly transforms method name" do
     expect(subject.send(:transform_method_name, "some_method_name")).to eq("SomeMethodName")
   end
+
+  describe '#base_url' do
+    subject { described_class.new.base_url }
+    context 'sets to test URL when in test mode' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.